Quickstore/pages-styles/wishlist.css

```diff
@@ -0,0 +1,51 @@
+.horizontal-card{
+    height: 15rem;
+    width: 26rem;
+    border: 1px solid var(--buybutton-color);
+    border-radius: 10px;
+    margin: 4rem;
+}
+
+.image-content img{
+    border-bottom-left-radius: 10px;
+    border-top-left-radius: 10px;
+}
+.wishlist-button button {
+    height: 2.5rem;
+    width: 10rem;
+    margin-top: 1.5rem;
+    font-size: 1rem;
+    font-weight: 500;
+}
+.move-wishlist{
+    background-color: var(--primary-color);
+}
+.remove-cart:hover{
+    background-color: var(--primary-color);
+}
+.final-price-tag{
+    grid-column: 2/3;
+    grid-row: 3/4;
+    max-width: 20rem;
+    margin-right: auto;
+    margin-left: 1rem;
```

Comment:
you can add this as variables later to keep the styling consistent for margin and padding

Reply:
ok i will consider